src/playground/App.js

```diff
@@ -245,6 +246,19 @@ const App = () => {
 		}
 	};
 
+	const onFixAll = () => {
+		const fixableMessages = messages.filter(message => message.fix);
+		if (fixableMessages.length > 0) {
+			const result = linter.verifyAndFix(text, optionsForLinter, {
+				fix: true,
+			});
+			setText(result.output);
+			storeState({
+				newText: result.output,
+			});
+		}
+	};
```

Comment:
```suggestion
const onFixAll = () => {
  const hasFixableMessages = messages.some(message => message.fix);
  if (!hasFixableMessages) return; 

  const result = linter.verifyAndFix(text, optionsForLinter, { fix: true });
  setText(result.output);
  storeState({ newText: result.output });
};
```

lets do Early return if no fixable messages.